osgar/logger.py

```diff
@@ -44,6 +44,8 @@
 TIMESTAMP_OVERFLOW_STEP = (1 << 32)  # in microseconds resolution
 TIMESTAMP_MASK = TIMESTAMP_OVERFLOW_STEP - 1
 
+logging.getLogger().setLevel(logging.ERROR)
```

Comment:
I do not like changing verbosity level in the middle of the code

Reply:
What is your preferred way to disable the warning about osgar logs directory?

Reply:
Oh, I see that I put it in the logger.py file while I intended to put it inside the test_logger.py. Would that do it?

Reply:
Done.